dashboard/app/views/levels/_single_multi.html.haml

```diff
@@ -3,6 +3,8 @@
 - use_tight_layout = data['layout'] == "tight" || tight_layout
 - force_wrap_layout = data['layout'] == "wrap"
 
+- multi_answer_characters = ("A".."Z").to_a
```

Comment:
In the basically not gonna happen chance that there are more than 26 options, what happens? Starts from the beginning?